packages/odyssey-react-mui/src/labs/SideNav.tsx

```diff
@@ -399,134 +401,144 @@ const SideNavItemLinkContent = ({
   );
 };
 
-const SideNavItemContent = ({
-  id,
-  label,
-  href,
-  target,
-  startIcon,
-  severity,
-  statusLabel,
-  endIcon,
-  onClick,
-  isSelected,
-  isDisabled,
-}: Pick<
-  SideNavItem,
-  | "id"
-  | "label"
-  | "href"
-  | "target"
-  | "startIcon"
-  | "severity"
-  | "statusLabel"
-  | "endIcon"
-  | "onClick"
-  | "isSelected"
-  | "isDisabled"
->) => {
-  const odysseyDesignTokens = useOdysseyDesignTokens();
-
-  const NavItemContentClickContainer = styled("div", {
-    shouldForwardProp: (prop) => prop !== "odysseyDesignTokens",
-  })(() => ({
-    display: "flex",
-    alignItems: "center",
-    width: "100%",
-    minHeight: "48px",
-    padding: `${odysseyDesignTokens.Spacing3} ${odysseyDesignTokens.Spacing4}`,
-    color: `${isDisabled ? odysseyDesignTokens.TypographyColorDisabled : odysseyDesignTokens.TypographyColorHeading} !important`,
-    "&:focus-visible": {
-      borderRadius: 0,
-      outlineColor: odysseyDesignTokens.FocusOutlineColorPrimary,
-      outlineStyle: odysseyDesignTokens.FocusOutlineStyle,
-      outlineWidth: odysseyDesignTokens.FocusOutlineWidthMain,
-      backgroundColor: odysseyDesignTokens.HueNeutral50,
-      textDecoration: "none",
+const SideNavItemContent = forwardRef<
```

Comment:
Added `forwardRef()` here, which caused a bunch of whitespace changes :(

Reply:
`forwardRef` plays dirty with the type system. You'll need to use `as` in those cases:
```
const MemoizedAutocomplete = memo(Autocomplete) as typeof Autocomplete;
```

If none of these components are memoized, they need to be. This is a great PR to add that.

Reply:
Another thing about `forwardRef`, if we expect them to pass in a `ref`, then we should also expect to limit their access to it. In this case, they should have no access to the underlying component through the `ref`.

We've used `useImperativeHandle` in the past for that. What it does is mask off external ref with an internal one. If they're not passing functions to the ref, it makes sense for us to manage that ourselves rather than forcing them to pass it in.

We can share that ref around these components with context so it "just works" when they're used together.